pkg/tfbridge/schema.go

```diff
@@ -341,6 +341,24 @@ func MakeTerraformInputs(
 	return makeTerraformInputsWithOptions(ctx, instance, config, olds, news, tfs, ps, makeTerraformInputsOptions{})
 }
 
+// Converts a single Pulumi property value into a plain go value suitable for use by Terraform.
+// This does not apply any defaults or other transformations.
+func makeSingleTerraformInput(
+	ctx context.Context, name string, val resource.PropertyValue, tfs shim.Schema, ps *SchemaInfo,
+) (interface{}, error) {
```

Comment:
Nit: Let's use `any` instead of `interface{}`.
```suggestion
) (any, error) {
```

Reply:
the rest of the file uses interface{} instead of any so I've opted for consistency here.